target/linux/x86/config-6.1

```diff
@@ -458,3 +458,5 @@ CONFIG_X86_VMX_FEATURE_NAMES=y
 CONFIG_XZ_DEC_BCJ=y
 CONFIG_XZ_DEC_X86=y
 CONFIG_ZLIB_INFLATE=y
+# CONFIG_DEVTMPFS_SAFE is not set
+# CONFIG_DM_AUDIT is not set
```

Comment:
Please add this in alphabetical order.
You can order it like this:
````
./scripts/kconfig.pl '+' target/linux/x86/config-6.1 /dev/null >target/linux/x86/config-6.1-new
mv target/linux/x86/config-6.1-new target/linux/x86/config-6.1
````
Maybe also `make kernel_oldconfig`works.

Reply:
Thanks for guidance. That was great. It's done, I have pushed the change or order.